### PR TITLE
Screen Reader and Keyboard Accesibility

### PR DESCRIPTION
**Pull request by msrafi**
Added Following Functionality

1. Screen Reader Attributes
2. Keyboard Accessibility
3. Focus Handling in Auto play mode

### CODE REVIEW

**Comment by gustavpursche** [comment]
@kenwheeler Are you somehow willing to merge this in? Is there something needed?

**Comment by kenwheeler** [comment]
I am I just need to review it

**Comment by kenwheeler** [comment]
@msrafi can you rebase?

**Comment by msrafi** [comment]
@kenwheeler <https://github.com/kenwheeler> Ohhh, created a new pull
request.

**Comment by msrafi** [comment]
@kenwheeler http://jsfiddle.net/1x8w9c3n/31/

File: slick/slick.js [hunk 1]
```diff
@@ -481,9 +481,9 @@
 
         _.updateDots();
 
-        if (_.options.accessibility === true) {
-            _.$list.prop('tabIndex', 0);
-        }
+        // if (_.options.accessibility === true) {
```
Comment (umpirsky) [inline slick/slick.js]: Should be removed?

Comment (msrafi) [inline slick/slick.js]: Yes. Let me create another fiddle with the latest code.

**Comment by msrafi** [comment]
Updated with later version: http://jsfiddle.net/equinoxs/1x8w9c3n/39/ 
About this one:
```javascript
     if (_.options.accessibility === true) {
              _.$list.prop('tabIndex', 0);
       }
```

This actually removed because, the tabIndex is handled inside _.activateADA method to give tabIndex to the active slide instead of all slides.

Fixed one more bug:
//Dont slide if the cursor is inside the form fields and arrow keys are pressed

**Comment by simeydotme** [comment]
looks like this was fixed by #1467 ?

**Pull request closed without merging**